downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md

```diff
@@ -144,6 +144,8 @@ To view bug fixes, see the [GitHub milestone for 2201.5.0 (Swan Lake)](https://g
 
 - Added parallel execution support for GraphQL resolvers.
 - Allowed adding a single interceptor without creating an array.
+- Skipped additional validation for unused operations in the GraphQL document.
+- Added support for exiting the listener when a panic occurs.
```

Comment:
This wording does not express the idea correctly. This sounds like a feature, but it is not. It's a change of behavior. @praneesha can we re-word this?

Reply:
Shall we say `Improved the listener behavior to exit when a panic occurrs.`?

Reply:
+1